web3rpc/rpc-specs/paths/subbridge/unsubscribeBridge.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: "3.0.2"
 info:
-  title: Klay Open API
+  title: Kaia Open API

```

Comment:
@Sotatek-TanHoang Could you make it uppercase?

Reply:
@kjeom it's done.